spec/models/financial_transaction_spec.rb

```diff
@@ -35,7 +35,7 @@
       expect(transaction).to eq(polymorphism_trans)
       expect(rent).to eq(polymorphism_trans.rental)
 
-      expect(transaction.amount).to eq((((rent.end_time.to_date - rent.start_time.to_date).to_i-1)*rent.item_type.fee_per_day)+rent.item_type.base_fee)
+      expect(transaction.amount).to eq(((rent.end_time.to_date - rent.start_time.to_date).to_i*rent.item_type.fee_per_day)+rent.item_type.base_fee)
```

Comment:
Not a fan of the change, but the critique has more to due with this spec, rather than your work. So, separate issue tangentially related to your work but out of scope for the time being. https://github.com/umts/probable-engine/issues/117